prereise/gather/griddata/hifld/data_process/transmission.py

```diff
@@ -57,6 +58,8 @@ def map_lines_to_substations_using_coords(
     :param int rounding: number of digits in coordinates rounded up to.
     :param bool drop_zero_distance_line: drop zero distance line or not, defaults to
         True.
+    :param float max_remap: maximum unit circle secant distance to map line endpoints. A
+        values of 1e-3 is equal to appropriately 5 miles.
```

Comment:
approximately?

Reply:
Yeah. I want to also refactor this so that the input can actually be miles or km, rather than this weird almost-radians distance measurement.

Reply:
Done